articles/azure-monitor/app/java-standalone-profiler.md

```diff
@@ -168,6 +187,12 @@ This value can be one of:
 - `profile`. Uses the `profile.jfc` jfc configuration that ships with JFR.
 - A path to a custom jfc configuration file on the file system, i.e `/tmp/myconfig.jfc`.
 
+`enableRequestTriggering` This will let know the profiler if JFR profiling should be triggered based on request configuration. 
```

Comment:
Suggest changing this to: "Whether JFR profiling should be triggered based on request configuration."